lambda/executor.js

```diff
@@ -174,6 +166,16 @@ const runInSeries = async({num, lambdaARN, lambdaAlias, payloads, preARN, postAR
     return results;
 };
 
+const handleLambdaInvocationError = (invocationResults, invocationMode, actualPayload, disablePayloadLogs) => {
```

Comment:
it looks like we could centralize this into utils.js, so it can be reused in `utils.invokeLambdaProcessor` as well

Reply:
Done